### PR TITLE
Martinh/ntt config update

### PR DESCRIPTION
**Pull request by martin0995**
### Description

Add details to the NTT configuration pages.

### Checklist

- [x] **Required** - I have added a label to this PR 🏷️
- [x] **Required** - I have run my changes through Grammarly
- [ ] If pages have been moved, I have created redirects in the `wormhole-mkdocs` repo

### CODE REVIEW

**Review by evgeniko — DISMISSED** [review]
lgtm

File: build/contract-integrations/native-token-transfers/configuration/rate-limiting.md [hunk 1]
```diff
@@ -7,22 +7,88 @@ description: Learn about rate limits in Wormhole NTT by configuring send/receive
...
 
-If a transfer is rate-limited on the source chain and queueing is enabled via `shouldQueue = true`, the transfer is placed into an outbound queue and can be released after the duration of the rate limit expires.
+If a transfer is rate-limited on the source chain and queueing is enabled via `shouldQueue = true`, the transfer is placed into an outbound queue and can be released after the rate limit expires.
 
 You can configure the following limits on every chain where NTT is deployed directly using the manager:
 
 - **Sending limit** - a single outbound limit for sending tokens from the chain
-- **Per-chain receiving limits** - the maximum receiving limit, which can be configured on a per-chain basis. For example, allowing `100` tokens to be received from Ethereum, but only 50 tokens to be received from Arbitrum
+- **Per-chain receiving limits** - the maximum receiving limit, which can be configured on a per-chain basis. For example, allowing `100` tokens to be received from Ethereum but only 50 tokens to be received from Arbitrum
```
Comment (eshaben) [inline build/contract-integrations/native-token-transfers/configuration/rate-limiting.md:15]:
```suggestion
- **Per-chain receiving limits** - the maximum receiving limit, which can be configured on a per-chain basis. For example, allowing 100 tokens to be received from Ethereum but only 50 tokens to be received from Arbitrum
```

File: build/contract-integrations/native-token-transfers/configuration/rate-limiting.md [hunk 2]
```diff
@@ -7,22 +7,88 @@ description: Learn about rate limits in Wormhole NTT by configuring send/receive
...
+        "outbound": "1000.000000000000000000",
+        "inbound": {
+            "Ethereum": "100.000000000000000000",
+            "Arbitrum": "50.000000000000000000"
+        }
+    }
+    ```
+
+     - `outbound` - sets the maximum tokens allowed to leave the chain
+     - `inbound` - configure per-chain receiving limits for tokens arriving from specific chains
```
Comment (eshaben) [inline build/contract-integrations/native-token-transfers/configuration/rate-limiting.md:38]:
```suggestion
     - **`outbound`** - sets the maximum tokens allowed to leave the chain
     - **`inbound`** - configures per-chain receiving limits for tokens arriving from specific chains
```

File: build/contract-integrations/native-token-transfers/configuration/rate-limiting.md [hunk 3]
```diff
@@ -7,22 +7,88 @@ description: Learn about rate limits in Wormhole NTT by configuring send/receive
...
 ## Queuing Mechanism
 
 When a transfer exceeds the rate limit, it is held in a queue and can be released after the set rate limit duration has expired. The sending and receiving queuing behavior is as follows:
 
 - **Sending** - if an outbound transfer violates rate limits, users can either revert and try again later or queue their transfer. Users must return after the queue duration has expired to complete sending their transfer
-- **Receiving** - if an inbound transfer violates rate limits, it is held in a queue. Users or relayers must return after the queue duration has expired to complete receiving their transfer on the destination chain
-    
+- **Receiving** - if an inbound transfer violates rate limits, it is in a queue. Users or relayers must return after the queue duration has expired to complete receiving their transfer on the destination chain
+
+Queuing is configured dynamically during each transfer by passing the `shouldQueue` parameter to the [transfer function](https://github.com/wormhole-foundation/native-token-transfers/blob/5e7ceaef9a5e7eaa13e823a67c611dc684cc0c1d/evm/src/NttManager/NttManager.sol#L171-L182){target=\_blank} in the `NttManager` contract.
```
Comment (eshaben) [inline build/contract-integrations/native-token-transfers/configuration/rate-limiting.md:90]:
```suggestion
Queuing is configured dynamically during each transfer by passing the `shouldQueue` parameter to the [`transfer` function](https://github.com/wormhole-foundation/native-token-transfers/blob/5e7ceaef9a5e7eaa13e823a67c611dc684cc0c1d/evm/src/NttManager/NttManager.sol#L171-L182){target=\_blank} in the `NttManager` contract.
```

**Review by eshaben — COMMENTED** [review]
Just a couple minor things 🙂

**Review by eshaben — APPROVED** [review]
LGTM!

**Pull request merged**